### ISSUE TITLE
Prevent websites with notification sounds (or any website) from pausing other websites playing media?

### ISSUE THREAD

**Issue by beyond9thousand** [NDevTK/AutoPause#20]
Discord notification sounds keep interrupting media playing in spotify, youtube etc.
Is there any way to bypass this?

**Comment by NDevTK** [comment]
To prevent multiple tabs from playing the correct logic is that the extension will pause media for tabs it has no control over.
There is the feature to "ignoredTabs" that I just never put on the firefox store.

Are you asking for an option so "Places to use" matches what media gets detected?

**Comment by NDevTK** [comment]
@beyond9thousand If that is what you wanted should be added as of https://github.com/NDevTK/AutoPause/actions/runs/1847625723

**Comment by beyond9thousand** [comment]
> @beyond9thousand If that is what you wanted should be added as of https://github.com/NDevTK/AutoPause/actions/runs/1847625723

Great! I'll take a look from my PC and get back to you @NDevTK

**Comment by beyond9thousand** [comment]
Yes, this solves it. Thank you so much for responding and making the tweak on this. Appreciated.

**Comment by NDevTK** [comment]
Its not on the store yet in version I linked theres now a "Ignore media from other tabs" option.

**Comment by beyond9thousand** [comment]
Ah, I see. I was wondering why it won't let me install the addon despite turning off signature required
![image](https://user-images.githubusercontent.com/77363063/154101544-5b3f67e5-9c63-45c8-ac89-49ab48cd21bb.png)

**Comment by NDevTK** [comment]
Added "Ignore other tabs" option in 2.7
Released on firefox store.
